src/Core/Domain/CartRule/QueryResult/EditableCartRuleMinimum.php

```diff
@@ -29,7 +29,6 @@
 namespace PrestaShop\PrestaShop\Core\Domain\CartRule\QueryResult;
 
 use PrestaShop\Decimal\DecimalNumber;
-use PrestaShop\PrestaShop\Core\Domain\Currency\ValueObject\CurrencyId;
 
 class EditableCartRuleMinimum
```

Comment:
```suggestion
class CartRuleMinimumForEditing
```
SHould we change all the query result naming with that convention? Since the query itself is `GetCartRuleForEditing` ?